components/session-service/server/server_functional_test.go

```diff
@@ -58,7 +58,7 @@ func TestMain(t *testing.T) {
 	mux := mux.NewRouter()
 	s := httptest.NewUnstartedServer(mux)
 	dexCerts := devDexCerts(t)
-	s.TLS = &tls.Config{
+	s.TLS = &tls.Config{MinVersion: tls.VersionTLS13,
```

Comment:
Suggest line break for readability/consistency.

Also, what is unique about this that it gets version 13 instead of 12?
(You have used version 12 everywhere except on 2 occurrences including this one.

```suggestion
	s.TLS = &tls.Config{
	  MinVersion: tls.VersionTLS13,
```